src/types/upgrades_info.v1/index.ts

```diff
@@ -91,6 +195,7 @@ export interface operations {
         };
         content: {
           'application/json': components['schemas']['Graph'];
+          'application/vnd.redhat.cincinnati.v1+json': components['schemas']['Graph'];
```

Comment:
Could this be the cause of the failing build jobs? There is also the same weird addition of `vnd.redhat.cincinnati.v1+` on line 262, which I haven't noticed anywhere else